[docs] Automatic user provisioning for MySQL (#33084)
gravitational/teleport: docs/pages/includes/database-access/auto-user-provisioning-intro.mdx

> Teleport can automatically create users in your database, removing the need for
> having to create each individual user account in advance, or using the same set
> of shared database accounts for all users.

Reviewer: ```suggestion
Teleport can automatically create users in your database, removing the need for
creating individual user accounts in advance or using the same set
of shared database accounts for all users.
```
Removing some repetition
